spec/features/assignments/new_spec.rb

```diff
@@ -13,15 +13,25 @@
     Timecop.return
   end
 
-  it 'returns the user to the appropriate index page' do
-    date_today = Date.new(2017, 4, 4)
-    start_date = Date.new(2017, 3, 31)
-    visit roster_assignments_url(roster, date: date_today)
-    visit new_roster_assignment_url(roster, date: start_date)
-    click_button 'Create'
-    expect(current_url)
-      .to eq roster_assignments_url(roster,
-                                    date: date_today
-                                              .beginning_of_month)
+  context 'returns the user to the appropriate index page' do
+    before :each do
+      @date_today = Date.new(2017, 4, 4)
```

Comment:
`let`s would be preferable to instance variables (here and in general).

Reply:
Right, I knew they were faster but for some reason I thought you couldn't edit an attribute of a `let` unless it was an instance variable. Not sure why though.